libs/ui-toolkit/modals/src/lib/ui-modal/simple-modal/simple-modal.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Inject, Input } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { SimpleModalData, SocialMediaArray } from './simple-modal.data';
+
+@Component({
+  selector: 'gor-simple-modal',
+  templateUrl: './simple-modal.component.html',
+  styleUrls: ['./simple-modal.component.scss']
+})
+export class SimpleModalComponent {
+  constructor(
+    public dialogRef: MatDialogRef<SimpleModalComponent>,
+
+   /**
+   * Inject needed interface SimpleModalData.
+   */
+    @Inject(MAT_DIALOG_DATA) public data: SimpleModalData
+  ) {}
+
+   /**
+   * Indicates default height value for modal.
+   */
+  @Input() public height = '400px';
+
+   /**
+   * Indicates default width value for modal.
+   */
+  @Input() public width = '500px';
+
+   /**
+   * Indicates default body text for modal.
+   */
+  @Input() public bodyText = 'Body content';
+
+   /**
+   * Indicates default title text for modal.
+   */
+  @Input() public titleText = 'Title';
+
+   /**
+   * Indicates default values for simple modal in this case specifically it is a the modal image.
+   */
+  @Input() public bodyImage = '/assets/img/comming-soon.svg';
+
+   /**
+   * Indicates if social media icon should be displayed.
+   */
+  @Input() public socialMedia = true;
```

Comment:
Since most modals we currently have do not display social media icon, a default value of `false` is more appropriate